invenio_records_resources/records/systemfields/files/manager.py

```diff
@@ -249,6 +249,32 @@ def delete_all(self, remove_obj=True, softdelete_obj=True, remove_rf=False):
                 remove_rf=remove_rf,
             )
 
+    def full_teardown(self):
+        """Hard delete bucket and Object versions associated with file manager."""
+        if self.enabled:
+            object_version_ids = [
+                entry.object_version_id for entry in self.entries.values()
+            ]
+            ObjectVersion.remove_many(object_version_ids, self.bucket)
```

Comment:
Here as well, I would go with just a delete query that removes everything in the bucket.